### DISCUSSION TITLE
Laravel 11 support

### DISCUSSION THREAD

**Discussion by tfortin** [coderflexx/laravel-ticket#54] [Ideas]
Hello,

Could you add support for Laravel 11 please? There is at least one failing dependancy to upgrade: `"illuminate/contracts": "^9.0|^10.0|^11.0"`.

Thanks

**Comment by ousid** [comment]
Hi, @tfortin 

Yes, sure. It took me a while to get time for it. Will add Laravel 11.x support tonight.

Thanks!

**Reply by aliaswpeu** [reply]
Hi @ousid,
I'm starting a new project, and I'd love to use this package. 
So, any progress on this ? :)
Cheers

**Reply by ousid** [reply]
You can use `v2.0.0` for Laravel 11.

Happy coding!!

**Reply by aliaswpeu** [reply]
Thank You!!
